### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by sukiletxe** [erkyrath/lectrote#92]
Issues encountered so far:
* The menus are not menus, they are dropdown buttons which are not keyboard-accessible (i. e. nothing happens when pressing alt or f10.).
* The text of the games is not read automatically (or rather, it is, but the text of the whole screen is read after the first command, and when the screen is cleared nothing is read (that second point may not be fixable)).

**Comment by sukiletxe** [comment]
(Actually, the second issue may be related to Quixe itself)

**Comment by erkyrath** [comment]
The menus are handled by the Chrome/Electron framework. I can't do anything about that. I believe https://github.com/electron/electron/issues/2504 is the filed bug covering it.

For the second point: Quixe has some ARIA markup, but I am not an expert on that and I can certainly believe that it can be improved. What I have is aria-live="polite", aria-atomic="false", aria-relevant="additions" on the story window. 

(As far I know, Lectrote's accessibility behavior should be exactly the same as Quixe running under Chrome. But it's not an area I've tested myself.)

**Comment by sukiletxe** [comment]
Yes, it's similar except for the first bit of the screen read when it 
shouldn't be. I mean: in Chrome, that bit is completely skipped, but in 
Lectrote that bit is read after entering the first command. I don't know 
why that happens. So, as this is not Lectrote's fault, feel free to 
close this if you want.
